automatminer/automl/gen_evolve_v2.py

```diff
@@ -0,0 +1,101 @@
+import random
+
+class NeuralNetOptimizer():
+    def __init__(self, X, y, model, retain = 5, random_select=0.05, mutate_chance=0.05, count=15):
+        self.random_select = random_select
+        self.mutate_chance = mutate_chance
+        self.retain = retain
+        self.X = X
+        self.y = y
+        self.Model = model
+        ACTIVATION = ["sigmoid", "tanh", "relu", "elu"]
+        OPTIMIZERS = ["sgd", "rmsprop", "adagrad", "adadelta", "nadam", "adamax", "adam"]
+        self.NN_PARAM_CHOICES = {"units": range(1, 1000), "hidden_layer_sizes": range(1, 3), "optimizer": OPTIMIZERS,
+                            "activation": ACTIVATION}
+        pop = self.create_population(count)
```

Comment:
As far as I can tell, you only run create_population once in this entire class, and it is not used as a method anywhere else in the code. It doesn't need to be its own method if it is only being used in init, and it is pretty cheap to run as far as i can tell.

On the other hand, if you expect create_population to be repeatedly used (i.e., to generate new populations using the same object) or it is expensive enough to be its own operation, it's ok to have its own method. But that does not seem like the use case here.